core/src/apps/cardano/address_id.py

```diff
@@ -0,0 +1,14 @@
+from micropython import const
+
+"""
+Address ID is used in the header (first byte)
+of each address in Shelley. Even Byron address
+has an implicit header.
+"""
+
+
+BASE_ADDRESS = const(0)
```

Comment:
can't we have this as enum? :thinking:

Reply:
I removed the file. Regarding enums - see comment above.